how-to.md

```diff
@@ -32,8 +32,9 @@ Please follow these instructions before your onboarding:
 
 ## Make sure a tenant meets the prerequisites to use Simeon
 
-- You must be operating on global Azure cloud (not [Government Community Cloud](https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/gcc)). The Azure Government cloud is not currently supported
-- Make sure the tenant has a valid Microsoft 365 license - all licensed SKUs are supported
+* Your tenant must be operating on global Azure or the [Government Community Cloud (GCC)](https://learn.microsoft.com/en-us/azure/azure-government/documentation-government-welcome)
```

Comment:
commercial Azure

Reply:
I used this because it is what we had before, and because MSFT refers to it as "global Azure", for example here: https://learn.microsoft.com/en-us/azure/azure-government/compare-azure-government-global-azure

I don't care either way.